### ISSUE TITLE
Null Reference Error Message with OnlineMeetings APIs

### ISSUE THREAD

**Issue by ayushiaks** [microsoftgraph/msgraph-sdk-serviceissues#114]
Hi, I've been trying to use APIs defined for getting online meeting information, which end up with this error:
`
{
    "error": {
        "code": "InternalServerError",
        "message": "Object reference not set to an instance of an object.",
        "innerError": {
            "date": "2021-10-14T07:10:16",
            "request-id": "74757b15-1444-4729-bce6-e689cfc0cfa9",
            "client-request-id": "74757b15-1444-4729-bce6-e689cfc0cfa9"
        }
    }
}
`

APIs that won't work:
1. https://graph.microsoft.com/beta/me/onlineMeetings/{meeting-id}/recording
2. https://graph.microsoft.com/beta/me/onlineMeetings/{meeting-id}/attendeeReport

The meeting id is correct, and /meetingAttendanceReport also works fine, these two don't work with v1.0 or beta.

**Comment by andrueastman** [comment]
Hey @ayushiaks,

Thanks for raising this issue. 

Are you able to replicate this issue using Graph Explorer?

**Comment by ayushiaks** [comment]
Yes, I've tried the same on Graph Explorer as well, I was able to reproduce it there.

**Comment by andrueastman** [comment]
Thanks for confirming this @ayushiaks.

I have raised this with the relevant workload team [here ](https://portal.microsofticm.com/imp/v3/incidents/details/267388845/home)and will update this issue with any feedback given.

**Comment by petrhollayms** [comment]
Thank you for reporting this issue. This appears to be an issue or limitation with the service APIs. Unfortunately, as the Microsoft Graph SDK team, we do not have ownership of the APIs that are causing you issues. We invite you to create a question about the service API to Microsoft Q&A and tagged with one of the [microsoft-graph-*] tags, that way it will get routed to the appropriate team for them to triage. 

[Microsoft Graph | Support](https://developer.microsoft.com/en-us/graph/support) or https://aka.ms/askgraph 


For now, we will close the issue on our side but feel free to open it in the relevant repository if you think the issue is specific to SDK. 
Please let us know if this helps! 

Note: We will close this repository on April 19, 2024.
